CodeGeneration/Sources/generate-swiftsyntax/templates/swiftsyntax/SyntaxVisitorFile.swift

```diff
@@ -0,0 +1,139 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+import SwiftSyntaxBuilder
+import SyntaxSupport
+import Utils
+
+let syntaxVisitorFile = SourceFile {
+  EnumDecl("""
+    /// The enum describes how the SyntaxVistor should continue after visiting
+    /// the current node.
+    public enum SyntaxVisitorContinueKind
+    """) {
+    EnumCaseDecl("""
+      /// The visitor should visit the descendents of the current node.
+      case visitChildren
+      """)
+    
+    EnumCaseDecl("""
+      /// The visitor should avoid visiting the descendents of the current node.
+      case skipChildren
+      """)
+  }
```

Comment:
I think this could be simplified to 
```suggestion
  EnumDecl("""
    /// The enum describes how the SyntaxVistor should continue after visiting
    /// the current node.
    public enum SyntaxVisitorContinueKind {
      /// The visitor should visit the descendents of the current node.
      case visitChildren
      /// The visitor should avoid visiting the descendents of the current node.
      case skipChildren
    }
    """)
```